src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,55 @@
+import styles from '../styles/ArticleCard.module.css';
+import {
+  TfiComment,
+  TfiAngleDown,
+  TfiAngleUp,
+  TfiTrash,
+} from 'react-icons/tfi';
+
+const ArticleCard = ({
+  title,
+  author,
+  created_at,
+  votes,
+  article_img_url,
+  comment_count,
+}) => {
+  return (
+    <li className={styles.card__articleContainer}>
+      <h2 className={styles.h2__articleTitle}>{title}</h2>
+      <h3 className={styles.h3__articleAuthor}>{author}</h3>
+      <h4 className={styles.h4__articleDate}>{created_at}</h4>
+      <div className={styles.div__articleImgWrapper}>
+        <img
+          className={styles.img__articleImg}
+          src={article_img_url}
+          alt={title}
```

Comment:
The alt text should be an alternate description of the image itself. I'm aware you don't know what each image could be, so feel free to make this something more generic, as the title of the article may not be tied to the image content